build/build/src/ci_gen.rs

```diff
@@ -639,14 +656,31 @@ fn benchmark(name: &str, command_line: &str, timeout_minutes: Option<u32>) -> Re
         wrap_expression(format!("true == inputs.{just_check_input_name}")),
     );
 
-    let mut benchmark_job = RunStepsBuilder::new(command_line)
-        .cleaning(CleaningCondition::Always)
-        .build_job(name, BenchmarkRunner);
-    benchmark_job.timeout_minutes = timeout_minutes;
-    workflow.add_job(benchmark_job);
+    for graal_edition in vec![graalvm::Edition::Community, graalvm::Edition::Enterprise] {
```

Comment:
Plain array literal can be iterated as well:
```suggestion
    for graal_edition in [graalvm::Edition::Community, graalvm::Edition::Enterprise] {
```

Reply:
Fixed in 4d0e5e6ef9f56e56554ce469e37ea6099096f685